### ISSUE TITLE
Feral Druid

### ISSUE THREAD

**Issue by Rasmutant** [Hekili/hekili#527]
**BEFORE SUBMITTING AN ISSUE:**

1.  Check to see if you are using the latest version of the addon.  If there is a newer alpha/beta/release version, see if your problem exists with the new version.

2.  Check to see if someone else has already opened an issue report here.  Please don't submit repeat reports.

**Describe the Bug/Issue**

**To Reproduce**
- Go to '...'
- Click on '....'
- Scroll down to '....'
- See error

**Expected behavior**

**Screenshots**

**WHEN SUBMITTING AN ISSUE:**

Please provide the following information.  Leaving it out may result in your ticket being ignored (because I won't have the information needed to explore your report).

1.  The information from the Issue Reporting tab ( /hekili > Issue Reporting ).  This provides your talent and gear information.  You can paste it to pastebin.com and provide a link here.

**PUT ISSUE REPORT LINK HERE:**

2.  If reporting a disagreement with the addon's recommendations, please take a snapshot when you observe the issue and provide the snapshot.  This is not a screenshot.  A snapshot can be generated by binding the Pause feature on the Toggles tab.  When you press Pause, a snapshot of how the addon decided on its CURRENT recommendations is generated and saved to the Snapshots tab.  Copy and paste this to pastebin.com and provide the link here.
 
**PUT SNAPSHOT LINK HERE:**

3.  If reporting an error (i.e., the addon freezes and/or disables itself), please install the BugSack and BugGrabber addons as they will enable you to paste the exact error message, with detail that will help me identify where the code error is located.  You can paste the error information here or use pastebin.com for that as well.

**PUT ERROR MESSAGE(S) HERE:**  https://pastebin.com/CLLxKhSa
build: v9.0.1-1.0.23
level: 53 (53)
class: DRUID
spec: feral

talents: balance_affinity
    brutal_slash
    mighty_bash
    moment_of_clarity
    sabertooth
    soul_of_the_forest
    wild_charge

pvptalents: none

covenant: none

azerite: none

essences: [none], none

sets/legendaries/artifacts: none

gear: ashjrakamas_shroud_of_resolve = 1
    blood_bog_shoulderpads = 1
    bookborrower_identification = 1
    chargestone_of_the_thunder_kings_court = 1
    courageous_aspirants_breeches = 1
    courageous_aspirants_helm = 1
    harmonias_chosen_belt = 1
    heart_of_azeroth = 1
    honorbound_outriders_boots = 1
    honorbound_outriders_bracers = 1
    honorbound_outriders_gloves = 1
    reverberating_silver_band = 1
    stonemasons_guild_band = 1
    waylight_tunic = 1
    zandalari_warstaff = 1

legendaries: none

itemIDs: 157883, 158075, 158129, 163309, 163437, 163438, 165660, 165678, 169223, 173320, 173331, 175626, 175704, 180444, 182682

settings: aoe = 3
    buffPadding = 0
    custom1Name = Custom 1
    custom2Name = Custom 2
    cycle = false
    cycle_min = 6
    damage = true
    damageDots = false
    damageExpiration = 3
    damagePets = false
    damageRange = 0
    debuffPadding = 0
    enabled = true
    gcdSync = true
    maxRefresh = 10
    maxTime = 33
    nameplateRange = 8
    nameplates = true
    package = Feral
    petbased = false
    potion = focused_resolve
    potionsReset = 20180919.1
    throttleRefresh = false
    throttleTime = false

toggles: cooldowns = false
    custom1 = false
    custom2 = false
    defensives = false
    essences = true
    interrupts = false
    mode = automatic
    potions = false

**Comment by Hekili** [comment]
Hi, I will need the other required information in the ticket as well.  Please provide, at the very least, the information from your Issue Reporting tab.

**Comment by Hekili** [comment]
It's wild, I didn't actually see this in-game at all, despite raiding on my Feral this week.

But I have a solution for this ready to go in the next release.  Thanks.

**Comment by Rasmutant** [comment]
I may have fixed my issue.  I deleted the mod during maintenance last night and re-installed.  Played this morning after I got the Hekili update and no issues.  Thanks


Sent from AT&T Yahoo Mail for iPhone


On Saturday, November 7, 2020, 8:22 AM, Hekili <notifications@github.com> wrote:




It's wild, I didn't actually see this in-game at all, despite raiding on my Feral this week.

But I have a solution for this ready to go in the next release. Thanks.
